ui/gulp/build.js

```diff
@@ -8,7 +8,7 @@ var $ = require('gulp-load-plugins')({
   pattern: ['gulp-*', 'main-bower-files', 'uglify-save-license', 'del']
 });
 
-const uglify = require('gulp-uglify-es').default;
+const uglify = require('gulp-uglify').default;
```

Comment:
Can you please explain why this change is required as it breaks my gulp build script

Reply:
The issue which you addressed seems to be an issue with your node-modules installed and not with the script.

Reply:
Earlier I made a clean install of node_modules but I was facing the error while running `gulp serve` that's why I pushed this change, if this thing is only with my machine then it will be better to skip this change.
![gulp-uglify](https://user-images.githubusercontent.com/31997208/59332693-7434f200-8d14-11e9-999f-84dec911766c.png)

Reply:
Please revert the change and also try installing that node module manually.

Reply:
Should we resolve this conversation @kmehant ?